mu-plugins/blocks/language-suggest/postcss/style.pcss

```diff
@@ -6,14 +6,28 @@
  */
 
 .wp-block-wporg-language-suggest {
-	background-color: #eff2ff;
-	color: #1e1e1e;
-	font-size: 12px;
+	--wp--custom--wporg-language-suggest--background: var(--wp--preset--color--blueberry-4, #eff2ff);
+	--wp--custom--wporg-language-suggest--color--text: var(--wp--preset--color--charcoal-1, #1e1e1e);
+	--wp--custom--wporg-language-suggest--font-size: var(--wp--preset--font-size--extra-small, 12px);
+	--wp--custom--wporg-language-suggest--spacing: var(--wp--preset--spacing--10, 10px);
```

Comment:
These are named like they can be set in theme.json, but they'll be overridden here if they are. You could put them in a `:root` block, so any theme.json settings would override it, but that would still be overridden by the prominent style (if you did, I'd also suggest renaming `--wp--custom--wporg-language-suggest--background` to `--wp--custom--wporg-language-suggest--color--background` so the object syntax would be correct).

Not a strong suggestion, I'm not sure it would be used if this was possible, but I tried it out since they're named matching the syntax.

```
"settings": {
	"custom": {
		"wporg/language-suggest": {
			"background": "var(--wp--preset--color--pomegrade-1)"
		}
	}
},
```

Reply:
Got it. 👍 

I've made some updates to look first for theme.json generated variables and then fallback in [92eb3f7](https://github.com/WordPress/wporg-mu-plugins/pull/607/commits/92eb3f71cb10c362fa4caf1e409481dc8deee212). Thoughts?

In terms of following object syntax, because I'm making use of the "custom" property, I have gone with the shortest name possible. If that breaks convention or isn't developer friendly, I don't mind updating. The css declarations were just getting long enough :).

Reply:
Example usage in theme.json:

```
"custom": {
  "wporg-language-suggest": {
     "background": "red"
  }
}
```

Reply:
I agree. I'll keep it local and merge. Thanks for the review.